src/api/status/public/css/animated-indication.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+.box {
+  position: relative;
+  height: 50px;
+  width: 250px;
+  background: transparent;
+  border-radius: 5px;
+}
+.box svg {
+  position: absolute;
+  height: 160px;
+  width: 160px;
+  left: 25%;
+  top: 50%;
+  transform: translate(-50%, -50%) rotate(-90deg);
+}
+.box svg circle {
+  fill: transparent;
+  stroke: rgb(252, 206, 3);
+  stroke-width: 3;
+  stroke-dasharray: 3;
+  stroke-dashoffset: 0;
+  transform-origin: center;
+  animation: rotate 1s linear infinite, stroke 3.9s linear infinite;
+}
+
+.box .counter {
+  position: absolute;
+  top: 60%;
+  left: 25%;
+  transform: translate(-50%, -50%);
+  width: 70%;
+  height: 30px;
+  color: #e6b800;
+  text-align: center;
+  overflow: hidden;
+}
+.box .counter span {
+  font-size: 10px;
+  line-height: 25px;
+  font-weight: bold;
+}
+h8 {
+  color: #000;
+  position: absolute;
+  width: 100%;
+  text-align: right;
+  top: 25%;
+  right: 2%;
+  letter-spacing: 1px;
+}
```

Comment:
I believe this HTML tag doesn't exist.

Reply:
it set the 'Staging' position.
![1639161277(1)](https://user-images.githubusercontent.com/55161917/145624455-59ee54ea-db64-496c-894a-f374609fd99c.png)

Reply:
What is `h8` tag? Can you provide a link to its documentation?